packages/suite/src/components/wallet/InputChecksumInfo.tsx

```diff
@@ -0,0 +1,75 @@
+import styled, { css } from 'styled-components';
+import { useTheme } from 'styled-components';
+import { TooltipSymbol } from '../suite';
+import { spacingsPx, borders } from '@trezor/theme';
+import { TranslationKey, Translation } from '../suite/Translation';
+import { useTranslation } from 'src/hooks/suite';
+import { Icon } from '@suite-common/icons/src/webComponents';
+
+const Wrapper = styled.div`
```

Comment:
Btw it would be useful to add screenshots to PRs like this

Reply:
Yes, It was just a copy of my PR that I created when I was outside of the org and couldn't run tests, to see that tests can run. 

In my initial PR there were screenshots, now I created a new one as a part of the org and will add screenshots there too.